plan_config.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
```

Comment:
I think both this and the other config file should be in a directory. Is that going to be difficult for packaging?

Reply:
Shouldn't be, we already copy them somewhere else.